### PR TITLE
in_exec: Ensure to sleep for prevending fork bomb

### PR DESCRIPTION
**Pull request by autopp**
Since 0.12.18,  `in_exec` dose not sleep `run_interval` when command or output parsing failed.
This causes the fork bomb.
This is prevended by moving `sleep @run_interval` to `ensure` clause.

(Sorry, the added test case may not be good...)

### CODE REVIEW

**Comment by tagomoris** [comment]
ping @repeatedly ?

**Comment by repeatedly** [comment]
I'm waiting patch update.

**Comment by repeatedly** [comment]
Ahh, wait. My comment isn't appear...

File: test/plugin/test_in_exec.rb [hunk 1]
```diff
@@ -130,4 +146,15 @@ def test_emit_regexp
...
+
+  def test_emit_with_invalid_script
+    d = create_driver invalid_json_config
+
+    d.run do
+      sleep 2
+    end
+
+    assert_equal true, d.emits.empty?
+    assert_equal true, File.read(@count_file).length.between?(1, 4)
```
Comment (repeatedly) [inline test/plugin/test_in_exec.rb]: For counting command execution, checking log lines seems enough like below:

https://github.com/fluent/fluentd/blob/e307297d88dc2890af28319faab30baead304988/test/plugin/test_in_forward.rb#L167

**Comment by autopp** [comment]
@repeatedly Thanks for your review and advice! I updated the test code.
Is it necessary to rebase commits?

**Comment by repeatedly** [comment]
> Is it necessary to rebase commits?

Not needed.

Thanks!

**Comment by autopp** [comment]
👍

**Pull request merged**